test/angular/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+// Test TypeScript and templates together to process inline templates.
+module.exports = {
+    overrides: [{
+        extends: '../../angular',
+        files: ['*.ts'],
+        parserOptions: {
+            project: './tsconfig.json',
+            // Resolve the project relative to this file's directory.
+            // https://stackoverflow.com/questions/64933543/parsing-error-cannot-read-file-tsconfig-json-eslint
```

Comment:
This seems like a useful comment. Can we add it to the other test configuration files for their `tsconfigRootDir` setting?